test/e2e/provisioning/all_e2e_test.go

```diff
@@ -362,46 +363,48 @@ func TestVsphereProvisioningE2E(t *testing.T) {
 		fmt.Sprintf("<< VSPHERE_CLUSTER >>=%s", vsCluster),
 	}
 	runScenarios(t, excludeSelector, params, VSPhereManifest, fmt.Sprintf("vs-%s", *testRunIdentifier))
+	runScenarios(t, excludeSelector, params, VSPhereDatastoreClusterManifest, fmt.Sprintf("vs-dsc-%s", *testRunIdentifier))
```

Comment:
Why are you running the test explicitly by calling the test runner? since you have already created a test inside `all_e2e_test.go` file, it would be better to create a job(in prow) which runs the test( please take a look here:  https://github.com/kubermatic/machine-controller/blob/master/.prow.yaml#L25-L45 ).

Reply:
I did not create a new test, the one you're probably referring to was already there and commented out

Reply:
Ok maybe I didn't explain it well. The new manifest that you have created to test the datastore cluster can be run as a separate test. Take a look at the aws cloud provider's tests. We have one test for default behavior and another test for `ebs` encryption. so instead of calling the runner n times the runner checks what to test and it will test it.

Reply:
ok, you' re proposing to have two distinct tests. I initially did that, then I grouped them to avoid some repetitions. It's true that having having dedicated tests bring some benefits, I can do that.